src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from django.conf import settings
+
+import httplib
+import json
+
+
+NLCD_MAPPING = {
+    11: ['water', 'Water'],
```

Comment:
Do these values line up with the values in the raster? Consider adding a comment.